<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LoginTidalDialog</class>
 <widget class="QDialog" name="LoginTidalDialog">
  <property name="windowTitle">
   <string>TIDAL Login</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="instructionsLabel">
     <property name="text">
      <string>1) Open TIDAL login in browser. 2) After login, copy the full redirected URL from the 'Oops' page. 3) Paste it below and click Complete Login.</string>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="openBrowserButton">
     <property name="text">
      <string>Open TIDAL Login in Browser</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="redirectUrlEdit">
     <property name="placeholderText">
      <string>Paste the redirected URL here...</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="hspacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="completeButton">
       <property name="text">
        <string>Complete Login</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="cancelButton">
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
 </ui>


